examples/example-next-static-export-sdk-2.x/src/lib/client.ts

```diff
@@ -10,6 +10,10 @@ const redirectUrl = process.env.NEXT_PUBLIC_FOUNDRY_REDIRECT_URL;
 checkEnv(url, "NEXT_PUBLIC_FOUNDRY_API_URL");
 checkEnv(clientId, "NEXT_PUBLIC_FOUNDRY_CLIENT_ID");
 checkEnv(redirectUrl, "NEXT_PUBLIC_FOUNDRY_REDIRECT_URL");
+const scopes = [
+    "api:read-data",
+    "api:write-data",
+];
```

Comment:
Should we wait until the ontologies namespace migration before exposing this?

Reply:
Well I've already tested the scenarios where the server side is `api:ontologies-X` and the client is `api:X-data` and it works since the endpoints check the post expansion operations. I think it is possible that we just roll the migrations out regardless of client-side changes. I will follow up with the teams on Monday to check on the status of these but I would prefer if information about the scopes is more explicit going forward given that hack week and DevCon is near.

Reply:
I would also like to point out that migrating these examples from `api:X-data` -> `api:ontologies-X` is part of the migration RFC (very final step) as well. So either way works for me

Reply:
We've did a quick discussion and we can actually push ahead with these changes here first but changing the example values to be `api:ontologies-read` and `api:ontologies-write` directly. This will not cause any regressions given that the applications would have either fully migrated to these new scopes and still preserve the `api:read-data` and `api:write-data` scopes which work